graylog2-server/src/main/java/org/graylog2/rest/models/system/responses/LocalesResponse.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * This file is part of Graylog.
+ *
+ * Graylog is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Graylog is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Graylog.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.graylog2.rest.models.system.responses;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.auto.value.AutoValue;
+import com.google.common.collect.ImmutableMap;
+
+import java.util.Arrays;
+import java.util.Locale;
+import java.util.function.Function;
+
+@JsonAutoDetect
+@AutoValue
+public abstract class LocalesResponse {
+    @JsonProperty("locales")
+    public abstract ImmutableMap<String, LocaleDescription> locales();
+
+    public static LocalesResponse create(Locale[] locales) {
```

Comment:
Locally I get the following entry (via `Locale.getAvailableLocales()`) among the expected ones:

    "und": {
      "language_tag": "und",
      "display_name": ""
    },


We should probably filter these out, not sure where they come from but they seem useless for the user.
According to [JDK docs](https://docs.oracle.com/javase/8/docs/api/java/util/Locale.html#toLanguageTag--) these seem to be representable in BCP 47.